pipemgr/pipemgr

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
```

Comment:
This file needs to be moved and altered. This shouldn't live inside of the root directory here as it's part of the source code and not an executable (set with #!/usr/bin/env python header). Only "scripts" should exist in the root path.

I'll leave a comment below and in the poetry config on how to make this install itself as a package and allow you turn the cli as a tool installed on the system.